server/internal/adapter/integration/api_asset.go

```diff
@@ -0,0 +1,196 @@
+		Keyword: nil,
+		Sort:    &sort,
+		Pagination: &usecasex.Pagination{
+			Before: nil,
+			After:  nil,
+			First:  lo.ToPtr(1000),
+			Last:   nil,
+		},
+	}
+
+	assets, pi, err := uc.Asset.FindByProject(ctx, id.ProjectID(request.ProjectId), f, op)
+	if err != nil {
+
+	}
+	itemList := lo.Map(assets, func(a *asset.Asset, _ int) integrationapi.Asset {
+		aa, err := ToAsset(a, uc.Asset.GetURL(a))
+		if err != nil {
+			return integrationapi.Asset{}
+		}
+		return *aa
+	})
+
+	return AssetFilter200JSONResponse{
+		Items:      &itemList,
+		Page:       lo.ToPtr(1),
+		PerPage:    lo.ToPtr(1000),
+		TotalCount: lo.ToPtr(pi.TotalCount),
+	}, nil
+}
+
+func (s Server) AssetCreate(ctx context.Context, request AssetCreateRequestObject) (AssetCreateResponseObject, error) {
+	uc := adapter.Usecases(ctx)
+	op := adapter.Operator(ctx)
+
+	f, err := ToFile(request.Body)
+	if err != nil {
+		return AssetCreate400Response{}, err
+	}
+
+	cp := interfaces.CreateAssetParam{
+		ProjectID: id.ProjectID(request.ProjectId),
+		File:      f,
+	}
+	a, err := uc.Asset.Create(ctx, cp, op)
+	if err != nil {
+		return AssetCreate400Response{}, err
+	}
+	aa, err := ToAsset(a, uc.Asset.GetURL(a))
+	if err != nil {
+		return AssetCreate400Response{}, err
+	}
+	return AssetCreate200JSONResponse(*aa), nil
+}
+
+func (s Server) AssetDelete(ctx context.Context, request AssetDeleteRequestObject) (AssetDeleteResponseObject, error) {
+	uc := adapter.Usecases(ctx)
+	op := adapter.Operator(ctx)
+	aId, err := uc.Asset.Delete(ctx, id.AssetID(request.AssetId), op)
+	if err != nil {
+		return AssetDelete400Response{}, nil
+	}
+	return AssetDelete200JSONResponse{
+		Id: &aId,
+	}, nil
+}
+
+func (s Server) AssetGet(ctx context.Context, request AssetGetRequestObject) (AssetGetResponseObject, error) {
+	uc := adapter.Usecases(ctx)
+	op := adapter.Operator(ctx)
+
+	a, err := uc.Asset.FindByID(ctx, id.AssetID(request.AssetId), op)
+	if err != nil {
+		return AssetGet400Response{}, err
+	}
+	aa, err := ToAsset(a, uc.Asset.GetURL(a))
+	if err != nil {
+		return AssetGet400Response{}, err
+	}
+	return AssetGet200JSONResponse(*aa), nil
+}
+
+func ToFile(multipartReader *multipart.Reader) (*file.File, error) {
+	for {
+		p, err := multipartReader.NextPart()
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			return nil, err
+		}
+
+		if p.FormName() != "file" {
+			if err := p.Close(); err != nil {
+				return nil, err
+			}
+			continue
+		}
+
+		var buf bytes.Buffer
+		tee := io.TeeReader(p, &buf)
+		defer p.Close()
+
+		return &file.File{
+			Content:     io.NopCloser(&buf),
+			Path:        p.FileName(),
+			Size:        getSize(tee),
+			ContentType: p.Header.Get("Content-Type"),
+		}, nil
+
+	}
+	return nil, errors.New("file not found")
+}
+
+func getSize(stream io.Reader) int64 {
+	buf := new(bytes.Buffer)
+	_, err := buf.ReadFrom(stream)
+	if err != nil {
+		return 0
+	}
+	return int64(buf.Len())
+}
+
+func ToAsset(a *asset.Asset, aUrl string) (*integrationapi.Asset, error) {
```

Comment:
```suggestion
func toAsset(a *asset.Asset, aUrl string) (*integrationapi.Asset, error) {
```

These functions are not used outside this package.